content/docs/user_guide/FAQ.md

```diff
@@ -35,6 +35,13 @@ If you’re having problems running a container with `runsc` it’s most likely
 to a compatibility issue or a missing feature in gVisor. See
 [Debugging](../debugging/).
 
+### When I run my container, docker fails and I see `error changing root filesystem: invalid argument` in the gVisor logs.
+
+The root filesystem for the container is likely located on a `tmpfs`, `ramfs`,
+or `initramfs` mount. These filesystem types cannot be used with
+[pivot_root](http://man7.org/linux/man-pages/man2/pivot_root.2.html). Put the
+root filesystem for your OCI bundle on a filesystem with a different type.
```

Comment:
Interesting, I wonder why this is the case?

Reply:
ramfs doesn't support pivot_root apparently. I should probably test it with tmpfs and ramfs but initramfs definitely doesn't work.

Reply:
So I think this actually happens when the root fs of the host is an initramfs and that forces the fs where pivot_root happens to be initramfs.  https://github.com/kubernetes/minikube/issues/3512

It's hard to recreate so I'll just close this.